single-research_project.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+use UCFResearchPublication\Common;
+
+$principle_investigator    = get_field( 'rp_principle_investigator', $post->ID );
+$investigators = get_field( 'rp_coprinciple_investigators', $post->ID );
+
+get_header(); the_post(); ?>
+
+<article class="<?php echo $post->post_status; ?> post-list-item">
+
+    <div class="container mt-4 mt-sm-5 mb-5 pb-sm-4">
+		<div class="row">
+			<div class="col-lg-8">
+				<?php the_content(); ?>
+            </div>
+
+            <?php if ( $principle_investigator || $investigators ) : ?>
+			<div class="col-lg-4">
+                <?php if ( $principle_investigator ) : ?>
+                    <h2 class="h5 heading-underline">Principle Investigator</h2>
+                    <dt>
+						<?php echo $principle_investigator->person_title_prefix; ?>
+						<?php echo $principle_investigator->post_title; ?>
+						<?php echo $principle_investigator->person_title_suffix; ?>
+					</dt>
+					<?php if ( $principle_investigator->person_jobtitle ) : ?>
+						<dd class="mb-0"><?php echo $principle_investigator->person_jobtitle; ?></dd>
+					<?php endif; ?>
+					<?php if ( $principle_investigator->person_phone ) : ?>
+						<dd class="mb-0">
+							<a href="tel:<?php echo $principle_investigator->person_phone; ?>">
```

Comment:
Phone number needs to have non-numeric characters removed from it before being printed in a `tel:` link value.